### ISSUE TITLE
About torch.nn.CTCLoss: Some problem 

### ISSUE THREAD

**Issue by lint39** [pytorch/pytorch#62557]
When I train a model using CTCLoss for several step, the loss can normally descent, but all the prediction are the first label. There seems to be something wrong.
![image](https://user-images.githubusercontent.com/51794023/127809019-ee01cec7-ac90-4bda-b9ee-e00caf65b6d4.png)
The valid accuracy 7.3% means that all the prediciton label are the same (ie. the first label) 
![image](https://user-images.githubusercontent.com/51794023/127809681-5efd801e-540d-413f-b868-57c74d56726a.png)

I make sure that I have followed the official docs, and the model can work normally for anthor task using Cross Entropy Loss。

Maybe someone knows what the problem is, and how to fix it? Thanks!

**Comment by jbschlosser** [comment]
Hey @lint39! I suggest posting this question to the [PyTorch forums](https://discuss.pytorch.org/), as you are more likely to get assistance there. Note that the issues here are reserved for issues within PyTorch itself. I'd also recommend providing some code snippets so others can assist you with the debugging process. Best of luck!
